compiler/src/iree/compiler/Codegen/Common/test/strip_config_info.mlir

```diff
@@ -0,0 +1,103 @@
+// RUN: iree-opt --split-input-file --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(builtin.module(func.func(iree-codegen-strip-config-info)))))' %s | FileCheck %s
+
+#config = #iree_gpu.lowering_config<{mma_kind = #iree_gpu.mma_layout<MFMA_F32_16x16x16_F16>, promote_operands = [0, 1], reduction = [0, 0, 64], subgroup_m_count = 2 : i64, subgroup_n_count = 2 : i64, workgroup = [64, 128, 0]}>
```

Comment:
You can trim this down significantly - as it is this is a "change detector" test in that any change to any attribute or op used in here will require someone to come fix the test. You should have an op count of 3 and the attributes should have the minimal stable configuration to pass verification.

The pass is 3 lines - the test should be correspondingly small.

Reply:
+1, we can write a small test case by hand that doesn't use the actual dumps from iree-opt. For example, setting `lowering_config` to a `StringAttr` should exercise the code just as well.